src/Microsoft.IdentityModel.Tokens/JsonWebKeyConverter.cs

```diff
@@ -33,7 +33,7 @@ namespace Microsoft.IdentityModel.Tokens
 {
     /// <summary>
     /// Converts a <see cref="SecurityKey"/> into a <see cref="JsonWebKey"/>
-    /// Supports: <see cref="RsaSecurityKey"/>, <see cref="X509SecurityKey"/> and <see cref=" SymmetricSecurityKey"/>.
+    /// Supports: converting to a <see cref="JsonWebKey"/> from one of: <see cref="RsaSecurityKey"/>, <see cref="X509SecurityKey"/>, and <see cref=" SymmetricSecurityKey"/>.
```

Comment:
It looks like we are missing a method that coverts ECDsaSecurityKey into a JsonWebKey.

Reply:
Yep, we don't have that yet. vNext

Reply:
Issue: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1208